packages/financial-templates-lib/src/clients/InsuredBridgeL2Client.ts

```diff
@@ -49,6 +49,9 @@ export class InsuredBridgeL2Client {
     return this.deposits[depositId.toString()];
   }
 
+  // TODO: consider adding a method that limits how far back the deposits will be returned from. In this implementation
+  // we might hit some performance issues when returning a lot of bridging actions
```

Comment:
empirically I've seen issues with Infura where it tries to fetch more than X blocks into the past. We should make sure that `index` sets the `startingBlockNumber` correctly.

Reply:
100p